Modules/Sources/WordPressUI/Views/Users/Views/UserDetailView.swift

```diff
@@ -0,0 +1,178 @@
+import SwiftUI
+
+struct UserDetailView: View {
+
+    private let userProvider: UserDataProvider
+    private let actionDispatcher: UserManagementActionDispatcher
+    let user: DisplayUser
+
+    @StateObject
+    var viewModel: UserDetailViewModel
+
+    @Environment(\.dismiss)
+    var dismissAction: DismissAction
+
+    init(user: DisplayUser, userProvider: UserDataProvider, actionDispatcher: UserManagementActionDispatcher) {
+        self.user = user
+        self.userProvider = userProvider
+        self.actionDispatcher = actionDispatcher
+        _viewModel = StateObject(wrappedValue: UserDetailViewModel(userProvider: userProvider))
+    }
+
+    var body: some View {
+        Form {
+            Section(Strings.nameSectionTitle) {
+                LabeledContent(Strings.roleFieldTitle, value: user.role)
+                LabeledContent(Strings.firstNameFieldTitle, value: user.firstName)
+                LabeledContent(Strings.lastNameFieldTitle, value: user.lastName)
+                LabeledContent(Strings.nicknameFieldTitle, value: user.handle)
+                LabeledContent(Strings.displayNameFieldTitle, value: user.displayName)
+            }
+
+            Section(Strings.contactInfoSectionTitle) {
+                LabeledContent(Strings.emailAddressFieldTitle, value: user.emailAddress)
+                if let website = user.websiteUrl {
+                    LabeledContent(Strings.websiteFieldTitle, value: website)
+                }
+            }
+
+            Section(Strings.aboutUserSectionTitle) {
+                LabeledContent(Strings.bioFieldTitle, value: user.biography ?? "")
+                if let profilePhotoUrl = user.profilePhotoUrl {
+                    LabeledContent(Strings.profilePictureFieldTitle) {
+                        UserProfileImage(size: CGSize(width: 96, height: 96), url: profilePhotoUrl)
+                    }
+                }
+            }
+
+            if viewModel.currentUserCanModifyUsers {
+                Section(Strings.accountManagementSectionTitle) {
+                    NavigationLink {
+                        UserChangePasswordView(viewModel: passwordChangeViewModel)
+                    } label: {
+                        Text(Strings.setNewPasswordActionTitle)
+                    }
+
+                    NavigationLink {
+                        // Pass this view's dismiss action, because if we delete a user, we want that screen *and* this one gone
```

Comment:
Both of these actions – "Set New Password" and "Delete User" should be shown as modal screens.
"Delete User" should be destructive.